### PR TITLE
US17701 – More form validation, resetting, and updating of the component

### PR DESCRIPTION
**Pull request by chriseallen**
**Problem**
After submitting a new email or password, the values remain in their input fields causing issues with toaster, the API, and security

**Solution**
Force the component to update the user's credentials after form submission. Then, remove the values from the form fields.

**Testing**
Local and INT

### CODE REVIEW

File: src/components/personalization/crds-profile-security/crds-profile-security.tsx [hunk 1]
```diff
@@ -205,33 +215,41 @@ export class CrdsProfileSecurity {
...
-        }
-        console.log(error.config);
-      });      
+          } else if (error.request) {
+            toastr.error('Something went wrong. Please try again.');
+            //console.log(error.request);
+          } else {
+            //console.log('Error', error.message);
+          }
+          //console.log(error.config);
```
Comment (j-brammer) [inline src/components/personalization/crds-profile-security/crds-profile-security.tsx]: no semicolons on line 229 and 230. I know JS oft doesn't care, but regardless, I just favor consistency.

I'm somewhat assuming the console.logs are there for debugging still?

File: src/components/personalization/crds-profile-security/crds-profile-security.tsx [hunk 2]
```diff
@@ -147,27 +147,35 @@ export class CrdsProfileSecurity {
...
+          {
+            headers: { Authorization: token }
+          }
+        )
+        .then(() => {
+          toastr.success('You have successfully updated your password.');
+          currentPasswordInput.value = ""
+          newPasswordInput.value = ""
+          confirmNewPasswordInput.value = ""
+        })
```
Comment (j-brammer) [inline src/components/personalization/crds-profile-security/crds-profile-security.tsx]: `;` 163, 164, 165

File: src/components/personalization/crds-profile-security/crds-profile-security.tsx (hunk 1 above)
Comment (chriseallen) [reply src/components/personalization/crds-profile-security/crds-profile-security.tsx]: I can remove them at this point I think

File: src/components/personalization/crds-profile-security/crds-profile-security.tsx (hunk 2 above)
Comment (chriseallen) [reply src/components/personalization/crds-profile-security/crds-profile-security.tsx]: `; `added

**Pull request merged**